wurst/systems/units/hostiles/Fawn.wurst

```diff
@@ -17,5 +45,40 @@ public class Fawn extends BabyAnimal
     override function getTypeId() returns int
         return unitTypeId
 
+    function checkForAcorn()
+        let fawn = getUnit()
+        if fawn.getOwner() != players[PLAYER_NEUTRAL_PASSIVE]
+            return
+
+        let pos = fawn.getPos()
+        let acorn = findNearestItem(pos, 300, i -> i.getTypeId() == ITEM_ACORN)
+        if acorn == null
+            return
+
+        let acornPos = acorn.getPos()
+        if acornPos.distanceTo(pos) > 50
+            fawn.issuePointOrderById(Orders.move, acornPos)
```

Comment:
use squared distance

Reply:
@Marsunpaisti then I have to square my real value as well, which I'm not inclined to do. I think `distanceSq` is fine when you're comparing two distances to one another but when you're comparing to a scalar it is less readable.